esp32s2-hal/ld/memory.x

```diff
@@ -21,10 +23,10 @@ STACK_SIZE = 8k;
 /* Specify main memory areas */
 MEMORY
 {
-  vectors_seg ( RX )     : ORIGIN = 0x40022000, len = 1k /* SRAM0 */
-  iram_seg ( RX )        : ORIGIN = 0x40022400, len = 128k-0x400 /* SRAM0 */
+  vectors_seg ( RX )     : ORIGIN = 0x40020000 + RESERVE_CACHES, len = VECTORS_SIZE /* SRAM0 */
+  iram_seg ( RX )        : ORIGIN = 0x40020000 + RESERVE_CACHES + VECTORS_SIZE, len = 192k - RESERVE_CACHES - VECTORS_SIZE /* SRAM0 */
 
-  dram_seg ( RW )        : ORIGIN = 0x3FFB0000 + RESERVE_DRAM, len = 192k - RESERVE_DRAM
+  dram_seg ( RW )        : ORIGIN = 0x3FFB0000 + RESERVE_CACHES + VECTORS_SIZE, len = 192k - RESERVE_CACHES - VECTORS_SIZE
```

Comment:
```suggestion
  dram_seg ( RW )        : ORIGIN = 0x3FFB0000 + RESERVE_CACHES, len = 192k - RESERVE_CACHES
```
The interrupt vectors area is already being considered on the offset in `iram0_reserved_for_data` output section when using `ORIGIN(RWTEXT)`, so it should be removed from here to avoid skipping it twice.

Reply:
that doesn't work - like with this change any example triggering interrupts will crash